spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -183,42 +183,42 @@
       expected["data"]["errors"] = [
         {
           "title" => "Invalid maths",
-          "detail" => "Pick an option for Maths",
+          "detail" => "Select an option for maths",
           "source" => {
             "pointer" => "/data/attributes/maths",
           },
         },
         {
           "title" => "Invalid english",
-          "detail" => "Pick an option for English",
+          "detail" => "Select an option for English",
           "source" => {
             "pointer" => "/data/attributes/english",
           },
         },
         {
           "title" => "Invalid sites",
-          "detail" => "You must pick at least one location for this course",
+          "detail" => "Select at least one location for this course",
           "source" => {
             "pointer" => "/data/attributes/sites",
           },
         },
         {
           "title" => "Invalid qualification",
-          "detail" => "You need to pick an outcome",
+          "detail" => "Select an outcome",
           "source" => {
             "pointer" => "/data/attributes/qualification",
           },
         },
         {
           "title" => "Invalid applications open from",
-          "detail" => "You must say when applications open from",
+          "detail" => "Enter when applications open from",
```

Comment:
What's the question that triggers this error message? 

I'm thinking that 'Enter the date when applications open' might be clearer, but I'd need to see how the question's phrased.

Reply:
@B-Bonner You can select from two options, one of which allows you to enter a specific date. Tough one…

<img width="500" alt="Screenshot 2021-07-06 at 10 27 36" src="https://user-images.githubusercontent.com/813383/124577028-ce95d380-de44-11eb-8ff7-7a9bb451ad00.png">

Reply:
I that case, I think:

- If the user doesn't select an option, it should read 'Select when applications will open'.
- If the user selects 'On a specific date', but doesn't enter the date, the error message should read 'Enter the date applications will open'.

Reply:
I think, at least for now, we’ll need one message that can cover both scenarios.

Reply:
Ok, then I think 'Select when applications will open'. Because its a radio button.